### ISSUE TITLE
KeyError when loading results

### ISSUE THREAD

**Issue by jeremyjbowers** [newsdev/elex#128]
```elex results 2012-11-06
2015-12-03 10:48:40,952 (INFO) elex (v0.0.42) : Getting results for election 2012-11-06
Whoa there, friend! There was an error:

You have not exported AP_API_KEY as an environment variable.```

This exception comes from `elex.cli.decorators` and is a sign of a `KeyError` of some variety.

**Comment by jeremyjbowers** [comment]
Ah, of course, a problem with the 2012 data and FIPS codes.

```2015-12-03 10:53:09,867 (INFO) elex (v0.0.42) : Getting results for election 2012-11-06
Traceback (most recent call last):
  File "/Users/jbowers/.virtualenvs/elex-loader/bin/elex", line 9, in <module>
    load_entry_point('elex==0.0.42', 'console_scripts', 'elex')()
  File "/Users/jbowers/.virtualenvs/elex-loader/src/nyt-ap-elections/elex/cli/app.py", line 155, in main
    app.run()
  File "/Users/jbowers/.virtualenvs/elex-loader/lib/python2.7/site-packages/cement/core/foundation.py", line 764, in run
    self.controller._dispatch()
  File "/Users/jbowers/.virtualenvs/elex-loader/lib/python2.7/site-packages/cement/core/controller.py", line 472, in _dispatch
    return func()
  File "/Users/jbowers/.virtualenvs/elex-loader/src/nyt-ap-elections/elex/cli/decorators.py", line 12, in decorated
    return fn(self)
  File "/Users/jbowers/.virtualenvs/elex-loader/src/nyt-ap-elections/elex/cli/app.py", line 113, in results
    self.app.render(self.app.election.results)
  File "/Users/jbowers/.virtualenvs/elex-loader/src/nyt-ap-elections/elex/parser/api.py", line 890, in results
    race_objs = self.get_race_objects(raw_races)
  File "/Users/jbowers/.virtualenvs/elex-loader/src/nyt-ap-elections/elex/parser/api.py", line 795, in get_race_objects
    return [Race(**r) for r in parsed_json['races']]
  File "/Users/jbowers/.virtualenvs/elex-loader/src/nyt-ap-elections/elex/parser/api.py", line 541, in __init__
    self.set_new_england_counties()
  File "/Users/jbowers/.virtualenvs/elex-loader/src/nyt-ap-elections/elex/parser/api.py", line 567, in set_new_england_counties
    results[ru.fipscode]['reportingunitname'] =  maps.FIPS_TO_STATE[ru.statepostal][ru.fipscode]
KeyError: u'9005'```

**Comment by eads** [comment]
ooh, that's funny. I mean not funny. So it's catching the wrong exception, eh? That decorator needs to be refined then.

**Comment by jeremyjbowers** [comment]
Yeah, this is probably my fault, too. I git-blamed myself and I was right!

**Comment by jeremyjbowers** [comment]
Fixed in sha d724cee278eb8ea7480924315c36fbfa7c0fddc0.
